lib/apiservers/engine/backends/container.go

```diff
@@ -752,7 +752,8 @@ func (c *Container) ContainerResize(name string, height, width int) error {
 // stop. Returns an error if the container cannot be found, or if
 // there is an underlying error at any stage of the restart.
 func (c *Container) ContainerRestart(name string, seconds *int) error {
-	defer trace.End(trace.Begin(name))
+	op := trace.NewOperation(context.Background(), "ContainerRestart - %s", name)
+	defer trace.End(trace.Begin(op.String()))
```

Comment:
FYI - you can pass the `op` to trace as a second parameter and it will include the opID in it's tracing exercise...
[see](https://github.com/vmware/vic/blob/master/lib/apiservers/portlayer/restapi/handlers/storage_handlers.go#L167) as an example